packages/docs/examples/va-slider/apiConfig.ts

```diff
@@ -0,0 +1,94 @@
+import { ManualApiOptions } from 'vuestic-ui/src/services/api-docs/ManualApiOptions'
+
+export const vaSliderOptions: ManualApiOptions = {
+  version: '1.3.0', // Not really, just for test.
```

Comment:
Please remove. I plan to add test page, where all documentation page features could be demoed.